BPCompatibleAlertControllerDemo/CompatibleAlertControllerDemo/ViewController.swift

```diff
@@ -9,52 +9,69 @@
 import UIKit
 
 class ViewController: UIViewController {
-    var alertController: BPCompatibleAlertController?
 
     override func viewDidLoad() {
         super.viewDidLoad()
         
+        var observerObject: NSObjectProtocol?
+        
+        let cleanupAfterAlertDismissed: (Void) -> Void =
```

Comment:
I do believe you're correct about the need to remove the observer. I'm just wondering if maybe there's a way we can do this automatically. What about storing the observers in the alertController and then in deinit we loop through and remove them?